### PR TITLE
577 Update OCCDS Vignette

### PR DESCRIPTION
**Pull request by millerg23**
Updated OCCDS vignette, and updated Occurence flag derivation in ADAE script.

### CODE REVIEW

File: vignettes/occds.Rmd [hunk 1]
```diff
@@ -389,8 +414,8 @@ adae <- tibble::tribble(
     7, "Alveolar proteinosis", NA_character_,  NA_integer_
 )
   
-adae <- derive_vars_query(dataset = adae , dataset_queries = queries)
-head(select(adae, USUBJID, ASTDTM, AETERM, AESEQ, AEDECOD, starts_with("SMQ05"), starts_with("CQ06")))
+adae_query <- derive_vars_query(dataset = adae1 , dataset_queries = queries)
+head(select(adae_query, USUBJID, ASTDTM, AETERM, AESEQ, AEDECOD, starts_with("SMQ05"), starts_with("CQ06")))
 ```
```
Comment (bundfussr) [inline vignettes/occds.Rmd:419]: I think we should use `dataset_vignette()` to display the datasets in the example. At the moment it is confusing that for the `queries` dataset `CQ01` and `SMQ02` are displayed but for the `adae_query` dataset `SMQ05` and `SMQ06`.

**Review by bundfussr — CHANGES_REQUESTED** [review]
The latest changes from `devel` are not included.

File: vignettes/occds.Rmd (hunk 1 above)
Comment (millerg23) [reply vignettes/occds.Rmd:424]: Done.

File: vignettes/occds.Rmd [hunk 2]
```diff
@@ -317,52 +337,57 @@ bds3 <- tibble::tribble(
  )
 derive_var_ontrtfl(
   bds3,
-  date = ADTM,
+  start_date = ADTM,
   ref_start_date = TRTSDTM,
   ref_end_date = TRTEDTM,
   filter_pre_timepoint = TPT == "PRE"
 )
```
[the remark this answers is not in the document]
Comment (millerg23) [reply vignettes/occds.Rmd]: I removed USUBJID and now ONTRTFL is visiible.

File: vignettes/occds.Rmd [hunk 3]
```diff
@@ -271,6 +285,12 @@ adae <- adae %>%
...
 ```
 
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(adae, 
+                 display_vars = vars(USUBJID, TRTSDT, TRTEDT, AESTDTC, ASTDT,
+                                     TRTEMFL))
+```
+
 To derive on-treatment flag (`ONTRTFL`) in an ADaM dataset with a single assessment date, we
 use `derive_var_ontrtfl()`.
```
[the remark this answers is not in the document]
Comment (millerg23) [reply vignettes/occds.Rmd:296]: fixed

File: vignettes/occds.Rmd [hunk 4]
```diff
@@ -271,6 +285,12 @@ adae <- adae %>%
   )
```
[the remark this answers is not in the document]
Comment (millerg23) [reply vignettes/occds.Rmd:285]: fixed.

File: vignettes/occds.Rmd [hunk 5]
```diff
@@ -18,77 +15,99 @@ knitr::opts_chunk$set(
...
 * [Read in Data](#readdata)
 * [Derive/Impute End and Start Analysis Date/time and Relative Day](#datetime)
 * [Derive Durations](#duration)
 * [Derive ATC variables](#atc)
-* [Derive Time Relative to Reference](#atirel)
-* [Derive Analysis Phase and Period](#phase_period)
 * [Derive Planned and Actual Treatment](#trtpa)
 * [Derive Date/Date-time of Last Dose](#last_dose)
 * [Derive Severity, Causality, and Toxicity Grade](#severity)
 * [Derive Treatement Emergent Flag](#trtfag)
```
[the remark this answers is not in the document]
Comment (millerg23) [reply vignettes/occds.Rmd]: Fixed.

**Review by bundfussr — APPROVED** [review]

**Pull request merged**